js/models/eventlist.js

```diff
@@ -0,0 +1,128 @@
+define([
```

Comment:
please move this to js/collections/eventList.js
- Collections get their own folder
- camelCase the fileNames